packages/entity-browser/src/modules/searchForm/sagas.js

```diff
@@ -27,6 +27,7 @@ export function* initialize({payload}) {
 
   const relationEntities = yield formDefinition.filter(searchField =>
     searchField.type === 'ch.tocco.nice2.model.form.components.simple.MultiSelectBox'
+    || searchField.type === 'ch.tocco.nice2.model.form.components.simple.SingleSelectBox'
```

Comment:
IE and some other browser dont support includes:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Check if babel transpiles "includes" or use IndexOf